Tasks/AzureRmWebAppDeployment/operations/KuduServiceUtility.ts

```diff
@@ -127,6 +127,54 @@ export class KuduServiceUtility {
         }
     }
 
+    public async zipDeploy(packagePath: string, appOffline?: boolean, customParameters?: Array<string>): Promise<void> {
+        try {
+            
+            if(tl.stats(packagePath).isDirectory()) {
+                let tempPackagePath = deployUtility.generateTemporaryFolderOrZipPath(tl.getVariable('AGENT.TEMPDIRECTORY'), false);
+                packagePath = await zipUtility.archiveFolder(packagePath, "", tempPackagePath);
+                tl.debug("Compressed folder " + packagePath + " into zip : " +  packagePath);
+            }
+
+            if(appOffline) {
```

Comment:
Set appOffline to false by default, as it is not enabled in case of Linux.

Reply:
done. thanks